File: EIDSSAVR/bv.common/Core/AvrException.cs
using System;
using System.Runtime.Serialization;

namespace bv.common.Core
{
    public class AvrException : ApplicationException
    {
        public AvrException()
        {
        }

        public AvrException(string message) : base(message)
        {
        }

        public AvrException(string message, Exception innerException) : base(message, innerException)
        {
        }

        protected AvrException(SerializationInfo info, StreamingContext context) : base(info, context)
        {
        }
    }
}

File: EIDSSAVR/vb/Shared/bv_common/Core/AvrException.vb
'Imports System.Runtime.Serialization
'
'Namespace Core
'    <Serializable()> _
'    Public Class AvrException
'        Inherits ApplicationException
'
'        Public Sub New()
'        End Sub
'
'        Public Sub New(ByVal message As String)
'            MyBase.New(message)
'        End Sub
'
'        Public Sub New(ByVal message As String, ByVal innerException As Exception)
'            MyBase.New(message, innerException)
'        End Sub
'
'        Public Sub New(ByVal info As SerializationInfo, ByVal context As StreamingContext)
'            MyBase.New(info, context)
'        End Sub
'
'    End Class
'End Namespace
